zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/distribution/CommandRedistributorTest.java

```diff
@@ -15,22 +15,25 @@
 import static org.mockito.Mockito.verify;
 
 import io.camunda.zeebe.engine.state.immutable.DistributionState.PendingDistributionVisitor;
+import io.camunda.zeebe.engine.state.routing.RoutingInfo;
 import io.camunda.zeebe.protocol.impl.record.value.distribution.CommandDistributionRecord;
 import org.junit.Before;
 import org.junit.Test;
 
 public class CommandRedistributorTest {
 
   private CommandDistributionBehavior behavior;
+  private RoutingInfo routingInfo;
 
   @Before
   public void setUp() {
     behavior = mock(CommandDistributionBehavior.class);
+    routingInfo = mock(RoutingInfo.class);
```

Comment:
:question: We have easy to use implementations, any reason to mock it here? If possible, let's avoid using mocks, since the behavior is harder to trace/assert in tests.